adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -602,9 +604,15 @@ public void acquireToken(String resource, String clientId, @Nullable String redi
     }
 
     /**
-     * acquireToken will start an interactive auth flow to acquire new tokens 
-     * with the requested claims. Bypasses token cache if promptbehavior is not AUTO or claims are passed. This overload uses new dialog based prompt. 
-     * It will create a handler to run the dialog related code. 
+     * acquireToken will authorize an end user to call the specified resource. 
+     * The minted access token will be sent to the {@link AuthenticationCallback}
+     * and can be used to call the specified protected resource. 
+     * Bypasses token cache if @param prompt is not AUTO or claims are passed.  
+     * This overload uses an 
+     * [AlertDialog](https://developer.android.com/guide/topics/ui/dialogs)
+     * in cases UI is required. This overload does not support any flow
```

Comment:
"in cases UI is required"

should it be "in cases [when/where] UI is required"